.gitignore

```diff
@@ -194,10 +194,11 @@ $RECYCLE.BIN/
 # Windows shortcuts
 *.lnk
 
-# End of https://www.gitignore.io/api/node,macos,windows,sublimetext,visualstudiocode
```

Comment:
Wait, why have you moved this?

Reply:
It was moved [lower](https://github.com/GalileoSchool/website/pull/11/files#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947R204), but I don't think belongs there because the following ignores are custom and not generated with gitignore.io

Reply:
are you sure? how much of the file WAS generated with gitignore.io? that's where it should be. from my understanding, everything before this point was generated